.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
want to add `.idea` to the top level gitignore?

Reply:
should be removed now and added to the top-level .gitignore